LICENSE.txt

```diff
@@ -1,3 +1,23 @@
+COPYRIGHT AND LICENSE NOTICE
+
+Copyright � 2008-2024: The CADET Authors, please see the AUTHORS and CONTRIBUTORS file.
```

Comment:
we dont have an AUTHORS file, only CONTRIBUTORS .. should we just refer to the CONTRIBUTORS file?

Reply:
good catch. I guess, maybe we had an AUTHORS file before. I will update that.